pom.xml

```diff
@@ -306,6 +310,11 @@
                 <artifactId>apicurio-registry-storage-kafkasql</artifactId>
                 <version>${project.version}</version>
             </dependency>
+            <dependency>
+                <groupId>io.apicurio</groupId>
+                <artifactId>apicurio-registry-serdes-common</artifactId>
```

Comment:
Mismatch between the artifact ID and the module's folder name. The folder is `serde-common` but the artifact id is `serdes-common`.